tests/monitoring/fixtures/monitor_request.py

```diff
@@ -43,32 +43,27 @@ def f(
             privkey = keys[1]
             privkey_non_closing = keys[0]
 
-        monitor_request = MonitorRequest(
+        bp = HashedBalanceProof(  # type: ignore
```

Comment:
Why do we need the `type: ignore` stuff here?

Reply:
Because I could get mypy to understand the optional `priv_key` parameter. I don't really want to ignore it.